packages/suite/src/components/suite/CollapsibleBox/index.tsx

```diff
@@ -22,6 +21,12 @@ const Wrapper = styled.div<Pick<Props, 'variant'>>`
             border-radius: 16px;
             box-shadow: 0 2px 5px 0 ${props.theme.BOX_SHADOW_BLACK_20};
         `}
+
+    ${props =>
+        props.disableShadow &&
```

Comment:
avoid adding extra props on universal components, you already styling CollapsibleBox in `pages/index.tsx` so let's just add `box-shadow: none;` there...